src/components/ShowHide/ShowHide.tsx

```diff
@@ -1,127 +0,0 @@
-/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Looks like `ShowHide` might be part of Project Overview? https://app.shortcut.com/czi-edu/story/190248/showhide cc @ifrost1